### DISCUSSION TITLE
Only downloading Headers and Avatars

### DISCUSSION THREAD

**Discussion by fragme11** [UltimaHoarder/UltimaScraper#1605] [Q&A]
It's only downloading the headers and avatars for all models after I made config changes so I didn't have to babysit and type 0 [Enter] over and over again.

I changed the following items in .profile\config.json 

  "settings": {
    "auto_site_choice": "onlyfans"

 "supported": {
    "onlyfans": {
      "settings": {
        "auto_profile_choice": "default",
        "auto_model_choice": true,
        "auto_media_choice": "0",
        ],
        "file_directory_format": "{site_name}\\{model_username}\\{media_type}",
        ],
        "overwrite_files": true,

It does this for all models....

Scrape Processing
Name: <removed>
Type: Profile
100%|| 3.18M/3.18M [00:00<00:00, 10.3MB/s]
Type: Stories
No Stories Found.
Type: Posts
Scrape Attempt: 1/100
Type: Archived Posts
Scrape Attempt: 1/100
Processing Scraped Posts
100%|| 313/313 [00:00<00:00, 2003.08it/s]
Processing metadata.
Finished processing metadata.
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
Renaming files.
100%|| 313/313 [00:00<00:00, 1648.12it/s]
Type: Archived
No Archived Found.
Type: Messages
Processing Scraped Messages
100%|| 20/20 [00:00<?, ?it/s]
Processing metadata.
Finished processing metadata.
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
Renaming files.
100%|| 20/20 [00:00<00:00, 1279.41it/s]
Scrape Completed

**Comment by Sox2017** [comment]
Did you figure this out
